coordinates/sites.json

```diff
@@ -8,9 +8,9 @@
         "latitude": 51.477811,
         "elevation_unit": "meter",
         "longitude": -0.001475,
-        "timezone": "Greenwich",
+        "timezone": "Europe/London",
         "aliases": [
-            "example_site"
+            "ROG"
```

Comment:
the ``"example_site"`` is important because it's used as an example site in some of the documentation/tutorials/etc, as the only "built-in" astropy observatory. So that has to stay in there to be consistent (but the "ROG" can/should also stay.)

Reply:
Ah I see. Was not familiar with the reason for that alias and thought it may have been left in by mistake.